docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+﻿version: "3.3"
```

Comment:
Why do we need a 4th docker-compose file?

Reply:
Removed